include/ignition/rendering/base/BaseMesh.hh

```diff
@@ -370,18 +370,16 @@ namespace ignition
 
     //////////////////////////////////////////////////
     template <class T>
-    const MeshDescriptor &BaseMesh<T>::GetMeshDescriptor() const
+    const MeshDescriptor &BaseMesh<T>::Descriptor() const
     {
       return this->meshDescriptor;
     }
 
     //////////////////////////////////////////////////
     template <class T>
-    void BaseMesh<T>::CopyMeshDescriptor(const MeshDescriptor &_copy)
+    void BaseMesh<T>::SetDescriptor(const MeshDescriptor &_desc)
     {
-      this->meshDescriptor.meshName = _copy.meshName;
-      this->meshDescriptor.subMeshName = _copy.subMeshName;
-      this->meshDescriptor.centerSubMesh = _copy.centerSubMesh;
+      this->meshDescriptor = _desc;
```

Comment:
Does this also copy the pointer to the mesh? As we discussed offline, I thought that we wanted to ignore storing/keeping track of this pointer

Reply:
yeah I took another look at the implementation and looks like the `common::Mesh` pointer is managed by the `common::MeshManager` and the pointer remains valid for the duration of the simulation session so it maybe ok. I don't have a strong opinion on this though

Reply:
As long as the pointer is valid, I am fine with storing it. Since it's a pointer, it doesn't take up much memory anyways. I say go ahead and merge it!